tool/dartformat.sh

```diff
@@ -44,8 +44,8 @@ if [[ $NULL_SAFETY == 1 ]]; then
     # $NULL_SAFETY_EXAMPLES/misc/lib/library_tour/async/stream.dart \
     # $NULL_SAFETY_EXAMPLES/misc/test/library_tour/core_test.dart \
     # $NULL_SAFETY_EXAMPLES/misc/test/library_tour/io_test.dart \
-    # $NULL_SAFETY_EXAMPLES/misc/lib/effective_dart/style_lib_good.dart \
-    # $NULL_SAFETY_EXAMPLES/misc/lib/effective_dart/usage_good.dart
+    $NULL_SAFETY_EXAMPLES/misc/lib/effective_dart/style_lib_good.dart \
```

Comment:
I think you might have to move up the entries you uncommented, so they're seen as part of the `dart format` command and not separate commands. At least, that's my guess as to why the build broke.

Reply:
Done.